Specs/DateTime/Japanese/DateParser.json

```diff
@@ -619,21 +619,22 @@
             "date": "2017-09-15"
           }
         },
-        "Start": 12,
-        "Length": 15
+        "Start": 2,
+        "Length": 6
       }
     ]
   },
   {
-    "Input": "私は3日火曜日に戻ります",
+    "Input": "私は第三火曜日に戻ります",
+    "Comment": "Translated from 'third thursday'.",
     "Context": {
       "ReferenceDateTime": "2017-09-27T00:00:00"
     },
     "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "3日火曜日",
```

Comment:
Would this case ('3日火曜日") never happen? Even in informal texting?
If it could happen, then we should also have a spec for it. can you add a new spec case in a new PR? This wouldn't block the current one.

Reply:
Grammatically '3日火曜日' just means "third day of that month which is Tuesday" and it's [already on the spec file](https://github.com/combacsa/Recognizers-Text/blob/c25e071fdfb2dce65121bab45da2c373b531151a/Specs/DateTime/Japanese/DateExtractor.json#L512). (27th day of that month which is Sunday). So I don't think it is necessary or probable to add a new spec case.

Reply:
And thanks for the reviewing and merging :)